src/test/java/org/springframework/data/r2dbc/repository/query/LikeEscaperTest.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2018-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.data.r2dbc.repository.query;
+
+import org.junit.Test;
+
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+/**
+ * @author Roman Chigvintsev
+ */
+public class LikeEscaperTest {
```

Comment:
Test names should be named `....UnitTests` or `....IntegrationTests` this applies to all tests.